docs/development-setup.md

```diff
@@ -196,22 +190,31 @@ projects:
 - Martin endpoints: [http://localhost:5002/tiles/accepting_stores/index.json](http://localhost:5002/tiles/accepting_stores/index.json) and [http://localhost:5002/tiles/accepting_stores/rpc/index.json](http://localhost:5002/tiles/accepting_stores/rpc/index.json). *The data shown on the map is fetched from a hardcoded url and is not using the data from the local martin!*
 - Map styles: [http://localhost:5002/map.html](http://localhost:5002)
 
-### Dumping and Restoring the Database
-
-```shell
-docker exec -ti <container_id> pg_dump -c -U postgres ehrenamtskarte > dump-$(date +%F).sql
-```
-
-To copy the dump to your local machine:
+### Common development tasks
 
-```shell
-rsync root@ehrenamtskarte.app:dump-2020-12-23.sql .
-```
+- Run Ktlint diagnostic: `./gradlew ktlintCheck`
+- Run Ktlint formatter: `./gradlew ktlintFormat`
+- Run Detekt diagnostic: `./gradlew detekt`
 
-To restore the dump
-```shell
-docker exec -i <container_id> psql ehrenamtskarte postgres < dump-$(date +%F).sql
-```
+### DB tasks
+
+- Clear the DB's contents: `./gradlew db-clear`
+- Run all migrations: `./gradlew migrate`
+- Import data from online stores: `./gradlew import`
+- Load developer sample data: `./gradlew db-import-dev`
+- Create a clean DB with sample data (all of the above steps): `./gradlew db-recreate`
```

Comment:
Could you replace steps 4 and 5 above with this command please? I think this is the way better way to set everything up now.